cmd/influxd/handler.go

```diff
@@ -28,15 +28,23 @@ func NewHandler() *Handler {
 
 // ServeHTTP responds to HTTP request to the handler.
 func (h *Handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	// FIXME: This is very brittle.  Refactor to have common path prefix
```

Comment:
That comment should actually be near line L42.  Was thinking of having all those handlers under something like `/data` so we don't have to explicitly list them in here.  When distributed queries are merged, I believe there is a new endpoint that will need to be added here so it doesn't break.